package.json

```diff
@@ -51,10 +51,15 @@
     "coffee-errors": "^0.8.6",
     "coffee-script": "~1.10.0",
     "csonschema": "^0.5.1",
+    "fs": "0.0.2",
     "glob": "^7.0.3",
+    "jsonlint": "^1.6.2",
     "mocha": "~2.5.1",
+    "node-rest-client": "^1.5.1",
+    "optimist": "^0.6.1",
     "mustache": "~2.2.1",
     "proxyquire": "^1.3.1",
+    "q": "^1.4.1",
```

Comment:
@cybertk  You are right. The packages was not added by me. I didn't notice that when I was rebasing the branch. I have removed those. 

@misfitdavidl If there is any issues of removing the packages which was added by you, please let me know.